README.md

```diff
@@ -6,18 +6,21 @@ Selamat datang kalian semua pada pejuang coding yang ingin berjuang untuk mengik
 
 Berikut merupakan orang-orang yang mengikuti academy ini :
 
-- [Adhi Muhammad Fahmi](https://github.com/adhiiimf) [AMF]
+- [Adhi Muhammad Fahmi](https://github.com/adhiiimf) [AMF 2019]
   > Sebagai mentor
-- [Iga Narendra Pramawijaya](https://github.com/IritaSee) [AND]
+- [Iga Narendra Pramawijaya](https://github.com/IritaSee) [AND 2018]
   > Sebagai mentor
-- [Muhammad Fakhri Putra Supriyadi](https://github.com/fakhrip) [FAI]
+- [Muhammad Fakhri Putra Supriyadi](https://github.com/fakhrip) [FAI 2017]
   > Sebagai mentor
-- [Dini Rizqi Amalia](https://github.com/dinudinni) [DIN]
+- [Dini Rizqi Amalia](https://github.com/dinudinni) [DIN 2020] 
   > Sebagai mentee
-- [Soediponegoro](https://github.com/Soedipo) [DIP]
+- [Soediponegoro](https://github.com/Soedipo) [DIP 2020]
   > Sebagai mentee
```

Comment:
I think its better to just leave the formatting as it was, because other mentee would need to adapt to this changes if this one has to change this way

Reply:
And for the possibility of having double assistant code could actually be solved by having their full name too, thats also the reason why i included the full name in the formatting